src/hooks/useVenueUserStatuses.ts

```diff
@@ -0,0 +1,32 @@
+import { DefaultUserStatus, User } from "types/User";
+
+import { USER_STATUSES } from "settings";
+import { WithId } from "utils/id";
+import { useRelatedVenues } from "./useRelatedVenues";
+import { useSovereignVenueId } from "./useSovereignVenueId";
+
+export const useVenueUserStatuses = (venueId?: string, user?: WithId<User>) => {
+  const { sovereignVenueId } = useSovereignVenueId({ venueId });
+
+  // Using currentVenueNG hook
+  const { currentVenue: venue } = useRelatedVenues({
```

Comment:
I think we should use a lighter hook here. Did you mean that in your comment? `currentVenueNG`

Reply:
https://github.com/sparkletown/sparkle/pull/1532#discussion_r657890368

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/71f5181e9d01672881376d7c0a0f0f0f917511eb